router.go

```diff
@@ -39,3 +59,8 @@ func main() {
 		known = routerConfig
 	}
 }
+
+func forever() {
+	for {
+	}
```

Comment:
I believe this is gonna spin on a core, not yielding to any other goroutines. might be a good idea to call [`runtime.Gosched()`](https://godoc.org/runtime#Gosched) inside the loop so others can do their work